cmd/collaborators/update.go

```diff
@@ -51,23 +51,11 @@ func NewUpdateCommand(clients *shared.ClientFactory) *cobra.Command {
 			return cmdutil.IsValidProjectDirectory(clients)
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if !clients.Config.WithExperimentOn(experiment.ReadOnlyAppCollaborators) {
-				cmd.Println()
-				cmd.Println(style.Sectionf(style.TextSection{
-					Emoji: "construction",
-					Text:  fmt.Sprintf("This command is under construction. Use at your own risk %s", style.Emoji("skull")),
-					Secondary: []string{
-						fmt.Sprintf("Bypass this message with the %s flag", style.Highlight("--experiment read-only-collaborators")),
-					},
-				}))
-				return nil
-			}
-
 			return runUpdateCommand(cmd, clients, args)
 		},
 	}
 
-	cmd.Flags().StringVarP(&updateFlags.permissionType, "permission-type", "P", "", "collaborator permission type: reader, owner")
+	cmd.Flags().StringVarP(&updateFlags.permissionType, "permission-type", "P", "", "collaborator permission type: [reader|owner]")
```

Comment:
suggestion: Updating the format of this to match the `add` command but not setting a default value.

<img width="762" height="570" alt="Image" src="https://github.com/user-attachments/assets/7d934b1f-e1b6-460b-9e30-3029b4738f76" />

```suggestion
	cmd.Flags().StringVarP(&updateFlags.permissionType, "permission-type", "P", "", fmt.Sprintf(
		"collaborator permission type\n(\"%s\" or \"%s\")",
		string(types.OWNER),
		string(types.READER),
	))```